evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -619,7 +618,7 @@ mod tests {
 
         let compiled = COMPILED.get("DebugLogs").expect("could not find contract");
         let (addr, _, _, _) =
-            evm.deploy(Address::zero(), compiled.bytecode.clone(), 0.into()).unwrap();
+            evm.deploy(Address::zero(), compiled.bin.clone().unwrap(), 0.into()).unwrap();
```

Comment:
there is also into_part(or_default) now

https://github.com/gakonst/ethers-rs/blob/bfdf70cbea9f96f98cb3ba6bba3db8a335e51aba/examples/contract_with_abi.rs#L23-L24